#global testerror "Err0439"

#test
{
    for a in {}
}